packages/mermaid/src/diagrams/packet/styles.ts

```diff
@@ -0,0 +1,29 @@
+import type { DiagramStylesProvider } from '../../diagram-api/types.js';
+import { log } from '../../logger.js';
+import type { PacketStyleOptions } from './types.js';
+
+export const styles: DiagramStylesProvider = (options: { packet?: PacketStyleOptions } = {}) => {
+  log.debug({ options });
+  return `
+	.byte {
+		font-size: ${options.packet?.byteFontSize ?? '10px'};
+	}
+	.byte.start {
+		fill: ${options.packet?.startByteColor ?? 'black'};
+	}
+	.byte.end {
+		fill: ${options.packet?.endByteColor ?? 'black'};
+	}
+	.label {
+		fill: ${options.packet?.labelColor ?? 'black'};
+		font-size: ${options.packet?.labelFontSize ?? '12px'};
+	}
+	.block {
+		stroke: ${options.packet?.blockStrokeColor ?? 'black'};
+		stroke-width: ${options.packet?.blockStrokeWidth ?? '1'};
+		fill: ${options.packet?.blockFillColor ?? '#efefef'};
+	}
```

Comment:
I think it would be better if we have a separate object that contains all default values for style options, then use `cleanAndMerge` on the passed parameter.